app/integration/runtime/src/main/java/io/syndesis/integration/runtime/handlers/SplitStepHandler.java

```diff
@@ -26,21 +27,50 @@
 import io.syndesis.common.model.action.StepAction;
 import io.syndesis.common.model.integration.Step;
 import io.syndesis.common.model.integration.StepKind;
-import io.syndesis.common.util.IOStreams;
 import io.syndesis.common.util.Json;
 import io.syndesis.common.util.SyndesisServerException;
 import io.syndesis.common.util.json.JsonUtils;
 import io.syndesis.integration.runtime.IntegrationRouteBuilder;
 import io.syndesis.integration.runtime.IntegrationStepHandler;
 import org.apache.camel.Exchange;
 import org.apache.camel.Expression;
+import org.apache.camel.TypeConverter;
 import org.apache.camel.builder.Builder;
 import org.apache.camel.model.ProcessorDefinition;
 import org.apache.camel.spi.Language;
 import org.apache.camel.support.ExpressionAdapter;
 import org.apache.camel.util.ObjectHelper;
 
 public class SplitStepHandler implements IntegrationStepHandler {
+
+    /* Types that need conversion to String in order to perform a split operation */
+    private enum AUTO_CONVERT_TYPES {
```

Comment:
```suggestion
    private enum AutoConvertedTypes {
```

PMD doesn't like it:
```
[INFO] PMD Failure: io.syndesis.integration.runtime.handlers.SplitStepHandler:47 Rule:ClassNamingConventions Priority:4 The enum name 'AUTO_CONVERT_TYPES' doesn't match '[A-Z][a-zA-Z0-9]*'.
```

Reply:
Argh! This has been a final static field before and I forgot to rename. Thanks! Lessons learned: Run the checks before checkin even if you think it is not necessary

Reply:
force pushed to this PR